scripts/build/setup/termux_setup_ldc.sh

```diff
@@ -0,0 +1,83 @@
+# shellcheck shell=bash disable=SC2155
+termux_setup_ldc_cross_config() {
+	local WIDER_TRIPLE=${TERMUX_LDC_TRIPLE/--/-.*-}
+	grep -F "$WIDER_TRIPLE" "$1" &>/dev/null && return;
+
+	cat << EOF >> "$1"
```

Comment:
If the indentation isn't critical, consider using `<<-`[^1] which will let you indent the heredoc more naturally.
```suggestion
	cat <<- EOF >> "$1"
```
[^1]: https://man.archlinux.org/man/bash.1#Here_Documents

Reply:
fixed